.config/.yamllint

```diff
@@ -3,13 +3,18 @@ extends: default
 
 rules:
   line-length:
-    max: 160
+    max: 300
```

Comment:
Why do we allow such long strings now?

If necessary, we can make an exception for individual files or lines, but not for the entire project.

Reply:
Feel free to change and tweak the affected lines. (I just did this to have a clean run initially with cleanup work being done later). 

I personally don't care about line length as I am wrapping in my editors. Hence I usually disable this check entirely. 

But yes, 300 is quite ambiguous and if the check should stay, a lower value with proper adoption is surely better.